autopilot/hostfilter.go

```diff
@@ -199,7 +199,7 @@ func isUsableHost(cfg api.AutopilotConfig, rs api.RedundancySettings, gc worker.
 		}
 
 		// perform gouging checks
-		if gougingBreakdown := gc.Check(&h.Settings, &h.PriceTable.HostPriceTable); gougingBreakdown.Gouging() {
+		if gougingBreakdown = gc.Check(&h.Settings, &h.PriceTable.HostPriceTable); gougingBreakdown.Gouging() {
```

Comment:
I'd probably do `gougingBreakdown = gc.Check(...)` and then `if gougingBreakdown.Gouging()`. Relying on an assignment within an `if` might not be obvious.